Source/WebKit/WPEPlatform/wpe/drm/WPEDisplayDRM.cpp

```diff
@@ -132,7 +137,7 @@ static std::unique_ptr<WPE::DRM::Connector> chooseConnector(int fd, drmModeRes*
 {
     for (int i = 0; i < resources->count_connectors; ++i) {
         WPE::DRM::UniquePtr<drmModeConnector> connector(drmModeGetConnector(fd, resources->connectors[i]));
-        if (!connector || connector->connection != DRM_MODE_CONNECTED || connector->connector_type == DRM_MODE_CONNECTOR_WRITEBACK || !connector->encoder_id)
```

Comment:
This is the part where I don't know if I am doing the right thing of this is because a buggy result from the imx6 DRM driver. In my results I connect to the fd successfully but the connector returns a `encoder_id == 0` what seems a bit extrange. However I google some implementations (e.g: [1](https://gist.github.com/dvdhrm/5083553#file-modeset-c-L61) ) where they manage this case and they read this 0 like _the connector is not currently bound to an encoder or if the encoder+crtc is already used by another connector_.